src/sentry/static/sentry/app/views/acceptOrganizationInvite.tsx

```diff
@@ -0,0 +1,219 @@
+import React, {MouseEvent} from 'react';
+import styled from 'react-emotion';
+import {browserHistory} from 'react-router';
+
+import {logout} from 'app/actionCreators/account';
+import {t, tct} from 'app/locale';
+import {urlEncode} from '@sentry/utils';
+import Alert from 'app/components/alert';
+import AsyncView, {AsyncViewProps, AsyncViewState} from 'app/views/asyncView';
+import Button from 'app/components/button';
+import ConfigStore from 'app/stores/configStore';
+import Link from 'app/components/links/link';
+import NarrowLayout from 'app/components/narrowLayout';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import space from 'app/styles/space';
+
+type InviteDetails = {
+  orgSlug: string;
+  needsAuthentication: boolean;
+  needs2fa: boolean;
+  needsSso: boolean;
+  existingMember: boolean;
+  ssoProvider?: string;
+};
+
+type State = AsyncViewState & {
+  inviteDetails: InviteDetails;
+  accepting: boolean | undefined;
+  acceptError: boolean | undefined;
+};
+
+class AcceptOrganizationInvite extends AsyncView<AsyncViewProps, State> {
+  getEndpoints(): [string, string][] {
+    const {memberId, token} = this.props.params;
+    return [['inviteDetails', `/accept-invite/${memberId}/${token}/`]];
+  }
+
+  getTitle() {
+    return t('Accept Organization Invite');
+  }
+
+  makeNextUrl(path: string) {
+    return `${path}?${urlEncode({next: window.location.pathname})}`;
+  }
+
+  handleLogout = async (e: MouseEvent) => {
+    e.preventDefault();
+    await logout(this.api);
+    window.location.replace(this.makeNextUrl('/auth/login/'));
+  };
+
+  handleAcceptInvite = async () => {
+    const {memberId, token} = this.props.params;
+
+    this.setState({accepting: true});
+    try {
+      await this.api.requestPromise(`/accept-invite/${memberId}/${token}/`, {
+        method: 'POST',
+      });
+      browserHistory.replace(`/${this.state.inviteDetails.orgSlug}/`);
+    } catch {
+      this.setState({acceptError: true});
+    }
+    this.setState({accepting: false});
+  };
+
+  get existingMemberAlert() {
+    const user = ConfigStore.get('user');
+
+    return (
+      <Alert type="warning" data-test-id="existing-member">
+        {tct(
+          'Your account ([email]) is already a member of this organization. [switchLink:switch accounts]?',
```

Comment:
I think this looks better capitalized:

```suggestion
          'Your account ([email]) is already a member of this organization. [switchLink:Switch accounts]?',
```